.github/workflows/publish.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Run Tests
         run: pnpm run cover 
 
-      - name: Publish Changeset and Package to NPM
+      - name: Publish packages
```

Comment:
i think you need to checkout the repo on line 9 with the admin account, take a look at the presentation repos pipeline

Reply:
same changes apply in the publish-dev

Reply:
Do I use the same admin account as presentation?

Reply:
Yes

Reply:
(Not sure)